GeometryTools/Rectangle.swift

```diff
@@ -195,7 +195,7 @@ public struct Rectangle: ConvexPolygonProtocol {
 
     /// - Returns: `Rectangle` geometrically equivalent to this one, with positive `height`
     /// and `width`.
-    public func normalized() -> Rectangle {
+    public var normalized: Rectangle {
```

Comment:
Then last thing, I promise. Because it's a `var` now, we can move it up above `vertices`.